src/main/java/org/mskcc/oncokb/curation/service/DrugQueryService.java

```diff
@@ -96,8 +99,13 @@ protected Specification<Drug> createSpecification(DrugCriteria criteria) {
             if (criteria.getId() != null) {
                 specification = specification.or(buildRangeSpecification(criteria.getId(), Drug_.id));
             }
-            if (criteria.getName() != null) {
-                specification = specification.or(buildStringSpecification(criteria.getName(), Drug_.name));
+            if (criteria.getName() != null && criteria.getName().getContains() != null) {
+                specification = specification.or((root, _ignore, cb) -> {
+                    return cb.like(
+                        cb.lower(root.get(Drug_.name).as(String.class)),
```

Comment:
MySQL `LIKE` operator should already compare ignoring case right? Does that mean we don't need `cb.lower(root.get(Drug_.name).as(String.class))`, instead we can trim to just `root.get(Drug_.name).as(String.class)`?